android/.gradle/2.4/taskArtifacts/cache.properties

```diff
@@ -0,0 +1 @@
+#Thu Oct 22 13:30:34 NZDT 2015
```

Comment:
We should add `android/.gradle` to the `.gitignore` and `git rm` the `.gradle` folder.